wallet/src/de/schildbach/wallet/ui/dashpay/DashPayViewModel.kt

```diff
@@ -132,6 +140,14 @@ class DashPayViewModel(application: Application) : AndroidViewModel(application)
         }
     }
 
+    fun getNextContactAddress(userId: String): Address {
+        var address: Address? = null
+        runBlocking {
+            address = platformRepo.getNextContactAddress(userId)
+        }
```

Comment:
It seems strange to have a `runBlocking` inside a view Model.  Should we use a liveData instead to get the next address for a particular user?